src/main/features/first-contact/guards/claimReferenceMatchesGuard.ts

```diff
@@ -1,17 +1,22 @@
 import * as express from 'express'
+import * as toBoolean from 'to-boolean'
+import * as config from 'config'
 import Claim from 'app/claims/models/claim'
 import ClaimStoreClient from 'app/claims/claimStoreClient'
 import { ErrorPaths } from 'first-contact/paths'
+import { OAuthHelper } from 'idam/oAuthHelper'
 
 const logger = require('@hmcts/nodejs-logging').getLogger('first-contact/guards/claimReferenceMatchesGuard')
 
+const oauthEnabled = toBoolean(config.get('featureToggles.idamOauth'))
+
 export default class ClaimReferenceMatchesGuard {
```

Comment:
I'd think about separating the implementations using OOP features here as well.

Reply:
hmm?

Reply:
Same as https://github.com/hmcts/cmc-citizen-frontend/pull/85/files#r145070497

Reply:
👌